packages/react-native-bridge/android/react-native-bridge/build.gradle

```diff
@@ -64,18 +64,18 @@ dependencies {
-        implementation "com.github.wordpress-mobile:react-native-slider:${readHashedVersion('../../../react-native-editor/package.json', '@react-native-community/slider', 'dependencies')}"
-        implementation "com.github.wordpress-mobile:react-native-get-random-values:${readHashedVersion('../../../react-native-editor/package.json', 'react-native-get-random-values', 'dependencies')}"
-        implementation "com.github.wordpress-mobile:react-native-masked-view:${readHashedVersion('../../../react-native-editor/package.json', '@react-native-community/masked-view', 'dependencies')}"
-        implementation "com.github.wordpress-mobile:react-native-gesture-handler:${readHashedVersion('../../../react-native-editor/package.json', 'react-native-gesture-handler', 'dependencies')}"
-        implementation "com.github.wordpress-mobile:react-native-screens:${readHashedVersion('../../../react-native-editor/package.json', 'react-native-screens', 'dependencies')}"
-        implementation "com.github.wordpress-mobile:react-native-safe-area-context:${readHashedVersion('../../../react-native-editor/package.json', 'react-native-safe-area-context', 'dependencies')}"
-        implementation "com.github.wordpress-mobile:react-native-reanimated:${readHashedVersion('../../../react-native-editor/package.json', 'react-native-reanimated', 'dependencies')}"
-        implementation "com.github.wordpress-mobile:react-native-prompt-android:${readHashedVersion('../../../react-native-editor/package.json', 'react-native-prompt-android', 'dependencies')}"
-        implementation "com.github.wordpress-mobile:react-native-webview:${readHashedVersion('../../../react-native-editor/package.json', 'react-native-webview', 'dependencies')}"
+        implementation "com.github.wordpress-mobile:react-native-svg:v9.13.7-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-video:5.0.2-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-linear-gradient:v2.5.6-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-slider:v3.0.2-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-get-random-values:v1.4.0-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-masked-view:v0.1.11-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-gesture-handler:1.10.1-wp-3"
+        implementation "com.github.wordpress-mobile:react-native-screens:2.9.0-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-safe-area-context:v3.2.0-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-reanimated:1.9.0-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-prompt-android:v1.0.0-wp-1"
+        implementation "com.github.wordpress-mobile:react-native-webview:v11.6.5-wp-1"
```

Comment:
We might need an alternative to hardcoding versions numbers. Maybe the function `readHashedVersion` can be updated to parse the versions from tarball URLs?

Reply:
Yeah, I think the best option would be to include a pattern for tarball URLs in that function 👍 .

Reply:
I've added the `readHashedVersion ` and add extra logic for extracting the version from the tarball URL in this commit:
https://github.com/WordPress/gutenberg/pull/34886/commits/db511824b7169a855e02fd9316637a6466b3398d

Reply:
I compiled the following results regarding the version extraction from the URL for each package:
```
=== readHashedVersion for react-native-svg
         fileName=react-native-svg-9.13.7-wp-1.tgz
         version=9.13.7-wp-1
=== readHashedVersion for react-native-video
         fileName=react-native-video-5.0.2-wp-1.tgz
         version=5.0.2-wp-1
=== readHashedVersion for react-native-linear-gradient
         fileName=react-native-linear-gradient-2.5.6-wp-1.tgz
         version=2.5.6-wp-1
=== readHashedVersion for @react-native-community/slider
         fileName=react-native-community-slider-3.0.2-wp-1.tgz
         version=react-native-community-slider-3.0.2-wp-1
=== readHashedVersion for react-native-get-random-values
         fileName=react-native-get-random-values-1.4.0-wp-1.tgz
         version=1.4.0-wp-1
=== readHashedVersion for @react-native-community/masked-view
         fileName=react-native-community-masked-view-0.1.11-wp-1.tgz
         version=react-native-community-masked-view-0.1.11-wp-1
=== readHashedVersion for react-native-gesture-handler
         fileName=react-native-gesture-handler-1.10.1-wp-3.tgz
         version=1.10.1-wp-3
=== readHashedVersion for react-native-screens
         fileName=react-native-screens-2.9.0-wp-1.tgz
         version=2.9.0-wp-1
=== readHashedVersion for react-native-safe-area-context
         fileName=react-native-safe-area-context-3.2.0-wp-1.tgz
         version=3.2.0-wp-1
=== readHashedVersion for react-native-reanimated
         fileName=react-native-reanimated-1.9.0-wp-1.tgz
         version=1.9.0-wp-1
=== readHashedVersion for react-native-prompt-android
         fileName=react-native-prompt-android-1.0.0-wp-1.tgz
         version=1.0.0-wp-1
=== readHashedVersion for react-native-webview
         fileName=react-native-webview-11.6.5-wp-1.tgz
         version=11.6.5-wp-1
```

Reply:
I see that most packages use the `v` prefix but others not 😭 , we would need to standardize this either by:
1. Generate the tarball with the corresponding Git tag.
2. Assure that all Git tags use the `v` prefix.

Reply:
A third option would be to add a new argument to the `readHashedVersion` function to determine if the prefix `v` should be used or note.

Reply:
I'm thinking even on a fourth 😄 , hardcode the `v` prefix on the Android dependencies entries.

Reply:
If we're going to update the Git tags, we may need to update the `Podspec`s to include the `v` as well. For example [this one](https://github.com/wordpress-mobile/react-native-reanimated/blob/4f4aa06ad3088215e6bd681a5060ee5ca8ea7bf0/RNReanimated.podspec#L17).

Reply:
> If we're going to update the Git tags, we may need to update the Podspecs to include the v as well. For example this one.

Good point, I think I'd go with option 4. However, from the tests I've done locally, the packages are fetched independently of adding the `v` prefix 😮 , looks like both formats can be used `vX.Y.Z` and `X.Y.Z`.

Reply:
> However, from the tests I've done locally, the packages are fetched independently of adding the v prefix 😮 , looks like both formats can be used vX.Y.Z and X.Y.Z.

I tested this in different packages and looks like this statement only applies to packages that use `vX.Y.Z` format.

**Package with format `vX.Y.Z` (example: [`react-native-webview`](https://github.com/wordpress-mobile/react-native-webview/releases)):**
- `implementation "${PACKAGE}:${VERSION}` (without `v`) ✅ 
- `implementation "${PACKAGE}:v${VERSION}` (with `v`) ✅

**Package with format `X.Y.Z` (example: [`react-native-reanimated`](https://github.com/wordpress-mobile/react-native-reanimated/releases)):**
- `implementation "${PACKAGE}:${VERSION}` (without `v`) ✅ 
- `implementation "${PACKAGE}:v${VERSION}` (with `v`) ❌

Reply:
Not really, all the CI jobs are passing so looks like it's not mandatory to use the `v` prefix here for packages with version format `vX.Y.Z`.

Reply:
I also checked on JitPack using the version with and without the prefix and both succeed:
<img width="590" alt="Screenshot 2021-09-27 at 15 47 44" src="https://user-images.githubusercontent.com/14905380/134921603-6dde0306-52be-48a8-a8a0-b74ec690d498.png">

Reply:
Cool, marking the thread as "resolved" and we can reopen of open a new one if needed 👍